plugin.rb

```diff
@@ -1079,6 +1082,19 @@ def run
         end
       end
     end
+
+    def add_stock_queries
+      json = JSON.parse(File.read(File.expand_path("../config/stock_queries.json", __FILE__)))
+      json["queries"].each do |params|
+        query = DataExplorer::Query.new
+        query.sql = params["sql"]
```

Comment:
@techAPJ Is there a way to assign these 5 properties in a more concise way? I tried using `assign_attributes` but that didn't seem to work.